src/bootstrap-table.js

```diff
@@ -2765,6 +2765,14 @@ class BootstrapTable {
     return this.scrollTo()
   }
 
+  getPageCount () {
+    return this.options.pagination ? this.options.totalPages : 1
+  }
+
+  getCurrentPage () {
+    return this.options.pagination ? this.options.pageNumber : 1
```

Comment:
The the getPageCount method would be also unnecessary, but as normal user its hard to figure out.

Reply:
[`pageNumber`](https://bootstrap-table.com/docs/api/table-options/#pagenumber) is an option in our API docs, but the `totalPages` is not. In addition, `pageSize` may also be used frequently. I don't think we need to add methods if you can get it from the options API.

Reply:
I understand what do you mean.
Lets wait for the opinion of @djhvscf, but then we should improve the documentation of the `getOptions` method.
Current it only says:
```
Return the options object.
```
Anybody who use the bootstrap-table the first time dont know what the options Object will contains.

Reply:
Another implementation is to return page info, such as:
```js
getPageInfo () {
  return {
    count: totalPage,
    current: pageNumber,
    size: pageSize
  }
}
```

Reply:
I think all the I formation is available from options object using the getOptions method. What we can do is improve the doc about that specific method instead of create new ones. But the getPageInfo method that @wenzhixin is proposing looks interesting. :)

Reply:
So if we sticky with the `getOptions` method we should also not implement the `getPageInfo` method. Because we should keep it uniform.

So lets stash that PR and lets improve the documentation of the existing method.